frontend/src/audio.html

```diff
@@ -0,0 +1,84 @@
+---
+pagination:
+  data: audios
+  size: 1
+  alias: audio
+permalink: "/audios/{{ audio.objectID | slug }}/"
+layout: base
+templateEngineOverride: md, njk, html
```

Comment:
This line (`templateEngineOverride`) seems unnecessary. Why would we override the template engine? Especially the value doesn't make sense. Or is there a reason?

Reply:
yes template override is not required, removed.